stdlib/influxdata/influxdb/from.go

```diff
@@ -25,6 +26,8 @@ func init() {
 		Return:   flux.TableObjectType,
 	}
 
+	flux.RegisterDependency(FromKind, "bucketlookup", dependencies.BucketLookup())
```

Comment:
Instead of named keys, should we have dependencies registered by the interface they implement? It would help to prevent name conflicts and would be decently easy. There's some precedent to doing this with `context.Context` and the `Set` method.

Reply:
truth: not sure how this got here.  was just playing with ideas.  will revert.